minari/dataset/minari_storage.py

```diff
@@ -44,23 +39,30 @@ def __init__(self, data_path: PathLike):
 
             self._combined_datasets = f.attrs.get("combined_datasets", default=[])
 
-            # checking if the base library of the environment is present in the environment
-            entry_point = json.loads(f.attrs["env_spec"])["entry_point"]
-            lib_full_path = entry_point.split(":")[0]
-            base_lib = lib_full_path.split(".")[0]
-            env_name = self._env_spec.id
-
-            try:
-                env = gym.make(self._env_spec)
-            except ModuleNotFoundError as e:
-                raise ModuleNotFoundError(
-                    f"Install {base_lib} for loading {env_name} data"
-                ) from e
-
-            self._observation_space = env.observation_space
-            self._action_space = env.action_space
-
-            env.close()
+            # ww will default to using the reconstructed observation and action spaces from the dataset
+            # and fall back to the env spec env if the action and observation spaces are not both present
+            # in the dataset.
+            if "action_space" in f.attrs and "observation_space" in f.attrs:
+                self._observation_space = deserialize_space(
+                    f.attrs["observation_space"]
+                )
+                self._action_space = deserialize_space(f.attrs["action_space"])
+            else:
+                # checking if the base library of the environment is present in the environment
```

Comment:
Pain to standardize them all but ideally we probably want most of the comments to be uppercase at the beginning and look consistent, not a huge deal but your commenting above was great so maybe just making sure it all looks good

Reply:
Fixed this instance